doc/extending/creating_a_numba_jax_op.rst

```diff
@@ -89,42 +88,76 @@ as :class:`CumsumOp`\ :class:`Op`. The difference lies in that the `mode` attrib
         c_axis = property(lambda self: np.MAXDIMS if self.axis is None else self.axis)
 
 `__props__` is used to parametrize the general behavior of the :class:`Op`. One need to
-pay attention to this to decide whether the JAX implementation should support all variants
+pay attention to this to decide whether the implementation should support all variants
 or raise an explicit NotImplementedError for cases that are not supported e.g., when
 :class:`CumsumOp` of :class:`CumOp("add")` is supported but not :class:`CumprodOp` of
 :class:`CumOp("mul")`.
 
 Next, we look at the :meth:`Op.perform` implementation to see exactly
 how the inputs and outputs are used to compute the outputs for an :class:`Op`
-in Python. This method is effectively what needs to be implemented in JAX.
+in Python. This method is effectively what needs to be implemented.
 
-Step 2: Find the relevant JAX method (or something close)
+Step 2: Find the relevant method in JAX/Numba/Pytorch (or something close)
 ---------------------------------------------------------
 
 With a precise idea of what the PyTensor :class:`Op` does we need to figure out how
-to implement it in JAX. In the best case scenario, JAX has a similarly named
+to implement it in JAX, Numba or Pytorch. In the best case scenario, there is a similarly named
 function that performs exactly the same computations as the :class:`Op`. For
 example, the :class:`Eye` operator has a JAX equivalent: :func:`jax.numpy.eye`
-(see `the documentation <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.eye.html?highlight=eye>`_).
+(see `the documentation <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.eye.html?highlight=eye>`_) and a Pytorch equivalent :func:`torch.eye` (see `documentation <https://pytorch.org/docs/stable/generated/torch.eye.html>`_).
 
-If we wanted to implement an :class:`Op` like :class:`IfElse`, we might need to
+If we wanted to implement an :class:`Op` like :class:`DimShuffle`, we might need to
 recreate the functionality with some custom logic.  In many cases, at least some
 custom logic is needed to reformat the inputs and outputs so that they exactly
 match the `Op`'s.
 
-Here's an example for :class:`IfElse`:
+Here's an example for :class:`DimShuffle`:
 
-.. code:: python
 
-   def ifelse(cond, *args, n_outs=n_outs):
-       res = jax.lax.cond(
-           cond, lambda _: args[:n_outs], lambda _: args[n_outs:], operand=None
-       )
-       return res if n_outs > 1 else res[0]
+.. tab-set::
+
+        .. tab-item:: JAX/Numba     
```

Comment:
This is not correct for Numba, can you leave it as a separate tab with [in progress] text (and open an issue) or check the source code of the Numba implementation if you want to do it correctly?

This probably applies to all the tabbed sections, no reason to combine jax and numba, and the pre-existing snippets were JAX specific

Reply:
I was adding a separate tab for numba and found this comment. Is there anything that should be changed in `numba_funcify_DimShuffle`?
https://github.com/pymc-devs/pytensor/blob/71592152992675274ac5d89d17a305a3a77764f9/pytensor/link/numba/dispatch/elemwise.py#L688-L690

Reply:
Not in the context of this PR, but we should open an issue to check if that's still a problem in the newer versions of numba. Could you do that?